app/views/branding_groups/list_navigate.html.erb

```diff
@@ -21,10 +21,13 @@
             <% end %>
           </div>
           <div class="collections-actions">
-            <% if @editable_branding_groups.include?(branding_group) %>
+            <% if branding_group.can_edit?(current_user) %>
               <%= scp_link_to "<i class='fas fa-search'></i> Info".html_safe, branding_group_path(branding_group), class: "btn btn-xs btn-info #{branding_group.name_as_id}-show" %>
               <%= scp_link_to "<i class='fas fa-edit'></i> Edit".html_safe, edit_branding_group_path(branding_group), class: "btn btn-xs btn-primary #{branding_group.name_as_id}-edit" %>
-              <%= scp_link_to "<i class='fas fa-trash'></i> Destroy".html_safe, branding_group_path(branding_group), method: :delete, class: "btn btn-xs btn-danger delete-btn #{branding_group.name_as_id}-delete" %>
+              <% if branding_group.can_destroy?(current_user) %>
```

Comment:
I like that you've added this here.  Unguarded deletes make me nervous though, is there an easy way to put this behind a confirm dialog?

Reply:
Yeah sure

Reply:
e6441de